flexget/plugins/clients/qbittorrent.py

```diff
@@ -205,6 +207,7 @@ def prepare_config(config):
         config.setdefault('use_ssl', False)
         config.setdefault('verify_cert', True)
         config.setdefault('label', '')
+        config.setdefault('tags', '')
```

Comment:
tags will be a list if the user enters it, so it's probably nicer to keep it the same type in the default case.
```suggestion
        config.setdefault('tags', [])
```